2.6.0/spec/orders.md

```diff
@@ -644,7 +650,7 @@ The date and time when the order is expected to be delivered. This is an estimat
 ### spSubscriptionId
 
 * Data format: [spSubscriptionId](../common/dataformats.md#spsubscriptionid)
-* Required in response, if set
+* Required
 
 ### subscriptionId
```

Comment:
This is probably out of scope, but could we rename `subscriptionId` to `coSubscriptionId`? So it's easier to understand what is what.

Reply:
Yes, that would be very good. Everything that avoids mistakes is good.